base_exception/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2011 Raphaël Valyi, Renato Lima, Guewen Baconnier, Sodexis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from . import wizard, models
+from .tests import test_tmp_model
```

Comment:
if you import it here w/out any condition is going to always be available.
Usualy I wrap the code of the test model inside an IF block. Like:
```python
testing = tools.config.get('test_enable') or os.environ.get('ODOO_TEST_ENABLE')
if testing:
    class Foo(...):
[...]
```
In this way is available only when running tests or on demand w/ the env var.

Reply:
Thanks for review. What's about import of ir.model.access.csv in the installation with testing = false ?

Reply:
don't care about that, remove ACL for this modeel. Is a testing model used for tests.

Reply:
@pedrobaeza actually if you want to simulate real usage is better to define a test model.
@mourad-ehm I just fixed an issue related to this w/ my approach here https://github.com/OCA/partner-contact/pull/526/commits/04a5bb7280238da000fd2dd764215cf257e904c8

Reply:
But travis test is fail because of ACL https://travis-ci.org/OCA/server-tools/jobs/309585651#L1032

Reply:
ping @simahawk. I added temp ACL to validate travis test. https://github.com/OCA/server-tools/pull/1025/commits/785be4dfa2ce767100f4fdbc3adb2e01b01c9956

Reply:
what you are linking is the std odoo warning and if you look into that build the LINT tests is green. In any case, I think you can disable MQT check for ACL on that file via `# pylint disable: $something`.  @moylop260?

Reply:
The tests folder is imported by default from odoo by standard `module/tests` is not required explicit import from `__init__.py`

Reply:
I'm seeing the last build of this PR but I saw a green result from travis.
What is the message line (link) to silent it?

Reply:
Travis is green because I added this fix https://github.com/OCA/server-tools/pull/1025/commits/785be4dfa2ce767100f4fdbc3adb2e01b01c9956 .
In fact travis require access.model for temporary models used for test.

Reply:
Hi @simahawk @moylop260 , I think we can merge this PR (travis and runbot are green) and make other PR if necessary to drop ACL data.

Reply:
fine for me. Just drop a line in ROADMAP so that we don't forget it. Thanks

Reply:
>The tests folder is imported by default from odoo by standard module/tests is not required explicit import from __init__.py

@moylop260 is this always true? I fear that is not imported unless you are specifically running tests for this module. [sale_exception PR](https://github.com/OCA/sale-workflow/pull/576) is failing because ["Model base.exception.test.purchase.line has no table"](https://travis-ci.org/OCA/sale-workflow/jobs/311944678#L1317)

Sounds like the models are registered at setup but they are not there anymore when running tests for sale_exception.